File: webApi/Models/Escala.cs
﻿using System;
using System.Collections.Generic;

namespace webApi.Models;

public partial class Escala
{
    public string Codigoaeropuertoescala { get; set; }

    public int? Vueloid { get; set; }

    public string Vueloorigen { get; set; }

    public string Ciudad { get; set; }

    public string Pais { get; set; }

    public virtual Vuelo Vuelo { get; set; }
}

File: AdminWebApp/src/app/Models/Escala.ts
import { Vuelo } from "./Vuelo";

export class Escala {
    codigoAeropuertoEscala: string;
    vueloId: number;
    vueloOrigen: string;
    ciudad: string;
    pais: string;
    vuelo: Vuelo;
  
    constructor(
      codigoAeropuertoEscala: string,
      vueloId: number,
      vueloOrigen: string,
      ciudad: string,
      pais: string,
      vuelo: Vuelo
    ) {
      this.codigoAeropuertoEscala = codigoAeropuertoEscala;
      this.vueloId = vueloId;
      this.vueloOrigen = vueloOrigen;
      this.ciudad = ciudad;
      this.pais = pais;
      this.vuelo = vuelo;
    }
  }